src/main/jbake/content/servlets.adoc

```diff
@@ -1,19 +1,19 @@
 type=page
 status=published
-title=Java Servlet Technology
+title=Jakarta Servlet Technology
 next=servlets001.html
 prev=jsf-ws007.html
 ~~~~~~
-= Java Servlet Technology
+= Jakarta Servlet Technology
 
 
 [[BNAFD]][[java-servlet-technology]]
```

Comment:
Fix the tag names.

Reply:
I intentionally didn't fix the tag names for now as that has the potential issue of breaking links. Since that's not visible, I think there should be no legal problems with it and my idea was to temporarily leave them and go back to them on a second pass. Do you think that's reasonable?

Reply:
Fixing the tags later is fine, although I think most of the links are to the BNAFD style name
so changing the tag names is unlikely to break links.